loopy/typing.py

```diff
@@ -75,6 +76,7 @@
 
 InameStr: TypeAlias = str
 InameStrSet: TypeAlias = frozenset[InameStr]
+ToInameStrSetConvertible: TypeAlias = "Iterable[InameStr] | InameStr"
```

Comment:
Some version of this seemed to be used quite a bit in `transform.iname`, so I just added one here. Good idea?